librosa/feature/inverse.py

```diff
@@ -195,26 +196,44 @@ def mfcc_to_mel(mfcc, n_mels=128, dct_type=2, norm='ortho', ref=1.0):
     ref : number or callable
         Reference power for (inverse) decibel calculation
 
+    lifter : number >= 0
+        If `lifter>0`, apply *inverse-liftering* (inverse cepstral filtering):
+        `M[n, :] <- M[n, :] / (1 + sin(pi * (n + 1) / lifter)) * lifter / 2`
 
     Returns
     -------
     M : np.ndarray [shape=(n_mels, n)]
         An approximate Mel power spectrum recovered from `mfcc`
 
+    Warns
+    --------
+    RuntimeWarning
+        overflow encountered in power or
+        divide by zero encountered during inverse-liftering.
 
     See Also
     --------
     mfcc
     melspectrogram
     scipy.fftpack.dct
     '''
+    if lifter > 0:
+        mfcc = mfcc.copy()
+        n_mfcc = mfcc.shape[0]
+        mfcc /= 1 + (lifter / 2) \
+            * np.sin(np.pi * np.arange(1, 1 + n_mfcc, dtype=mfcc.dtype) / lifter)[:, np.newaxis]
 
-    logmel = scipy.fftpack.idct(mfcc, axis=0, type=dct_type, norm=norm, n=n_mels)
+        # avoid division of zero by zero errors
+        mfcc[np.isnan(mfcc)] = 1.
```

Comment:
is this really needed? can't you catch this before applying the division?
i would prefer to avoid multiple sequential assignments on the same array. they tend to be error-prone. ideally, the output is computed in a single pass.

Reply:
This is not my favorite line and I agree with your concern. I replaced it by the following `np.where(mfcc == 0, np.finfo(float).eps, mfcc) ` which should help avoid the zero by zero division errors.

Reply:
This change resulted in a failed build :( any suggestions on how to fix this?

Reply:
I fixed it.